contracts/interfaces/USSSpokePoolInterface.sol

```diff
@@ -101,6 +123,19 @@ interface USSSpokePoolInterface {
         bytes updatedMessage
     );
 
+    event USSExecutedRelayerRefundRoot(
```

Comment:
This is pure bikeshedding, but I am kinda partial to `<verb>USS<noun>()`. For example: `ExecutedUSSRelayerRefundRoot()`.

Reply:
I'm good with that